.github/pull_request_template.md

```diff
@@ -4,20 +4,42 @@
 
 Fixes #
 
-# 👩🏻‍💻 What does this PR do? 
- <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
+# 👩🏻‍💻 What does this PR do?
 
-# 🧪 How has/should this change been tested? 
-<!-- Explain how to setup for testing here if it is not already obvious, and how you've tested this PR. -->
+<!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
 
 ## 💌 Any notes for the reviewer?
-<!-- eg. Do you have any specific questions for the reviewer? Is there a high risk/complicated change they should focus on? If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here.-->
 
-## 📃 Documentation
+<!--
+eg. Do you have any specific questions for the reviewer? Is there a high risk/complicated change they should focus on? If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here.
+Anything half cooked but going to be finished off in a different PR?
+-->
+
+# 🧪 Testing
+
+<!-- Explain how to setup for testing here if it is not already obvious, and steps you've taken to test this PR. -->
+
+## Setup
+
+<!-- Explain how to setup for testing here if it is not already obvious. -->
+
+- [ ] _(e.g.)_ Central Sync server with 1 Legacy Desktop remote site and 1 OMS remote site running this PR
+- [ ] _(e.g.)_ This sample datafile:
+
+## Tests
+
+<!-- Explain the steps you'd take to test the changes of this PR manually -->
+
+- [ ] _(e.g.)_ Open a requisition with some lines
+- [ ] _(e.g.)_ Make a couple invoices supplying some amount of those lines
+- [ ] _(e.g.)_ Review that "issued" column is the sum of the amount already issued in invoices for this requisition
+
+# 📃 Documentation
```

Comment:
I'm not finding this section as useful as I'd hoped when I wrote it originally!
It is another section that requires selecting and deleting stuff *every time* I do a PR and that's annoying. what about checkboxes?

* [ ] **Part of an epic**: documentation will be completed for the feature as a whole
* [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
* [ ] **These areas should be updated or checked**: <!-- _(e.g.)_ New `issued` column in `Requisitions` indicates stock quantity already in shipments -->
  1. 
  2.     


( having the e.g. bit in the list means that I have to delete it every time, could it just be a comment instead? )

Reply:
Agree again, I am the same I hate leaving cruft of empty sections when they're not applicable and it's tedious removing it - but it's a bother when they _should_ have been filled and been ignored... maybe just reminding people when it happens is better but tedious in another way 😁.

In the issues the checkboxes bothered me because they get jumbled up with the progress indicator on issues (github interprets them as subtasks), but on PRs yea could work. That said there is some doubling up with labels recently added for managing documentation as well.